app/kubernetes-ingress-controller/1.0.x/concepts/ingress-versions.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Ingress v1 and v1beta1 differences
+---
+
+## Introduction
+
+Kubernetes 1.18 introduced a new v1 API for the [Ingress resource][kubernetes-ingress-doc].
```

Comment:
```suggestion
Kubernetes 1.18 introduced a new `networking.k8s.io/v1` API for the [Ingress resource][kubernetes-ingress-doc].
```

this is because Kubernetes API versions are always qualified like this; `v1` suggests it being part of core.